html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <p>Email
+      <br/>
```

Comment:
You should avoid using the <br/> element as much as possible. If you want your text to appear on a new line and not next to other elements, you should be using a <div> instead with a <p> tag inside.